Frameworks/UIKit/UIDatePicker.mm

```diff
@@ -814,6 +813,8 @@ static void resetPickerPositions(UIDatePicker* self) {
  @Status Interoperable
 */
 - (void)setMinimumDate:(NSDate*)date {
+    [date retain];
+    [_minimumDate release];
     _minimumDate = date;
```

Comment:
@DHowett-MSFT May have other comments on the general convention we want here?

---
In reply to: [74988055](https://github.com/Microsoft/WinObjC/pull/883#discussion_r74988055) [](ancestors = 74988055,74986350,74983076)